risedev.yml

```diff
@@ -964,6 +964,22 @@ profile:
       - use: kafka
         persist-data: true
 
+  ci-1cn-1fe-user-kafka-with-recovery:
```

Comment:
There seems no other occurrence of `ci-1cn-1fe-kafka-with-recovery`. Can we also remove that?

Reply:
done https://github.com/risingwavelabs/risingwave/pull/16977/commits/a8b29f16fb200566b023aa5b9af6b292ea7e756e